features/page_objects/provider_page.rb

```diff
@@ -0,0 +1,34 @@
+require_relative 'sections/checkbox_fee_section'
```

Comment:
does not look like you are using this, no call to
```
CheckboxFeeSection
```

**Not a blocker**
You could, as a refactor, write a `CheckboxSection`, and have `CheckboxFeeSection` inherit from that and use the new CheckboxSection here, as below
```
section :fee_scheme_checkboxes, CheckboxSection, "div#fee-scheme-options"
```